tests/functional/cylc-combination-scripts/09-vr-icp-now.t

```diff
@@ -26,14 +27,13 @@ WORKFLOW_ID=$(workflow_id)
 
 cp -r "${TEST_SOURCE_DIR}/${TEST_NAME_BASE}/flow.cylc" .
 
-run_ok "${TEST_NAME_BASE}-vip" \
-    cylc vip . \
-        --workflow-name "${WORKFLOW_ID}" \
-        --no-detach \
-        --no-run-name
+run_ok "${TEST_NAME_BASE}-vip" cylc vip . \
+    --workflow-name "${WORKFLOW_ID}" \
+    --no-detach \
+    --no-run-name \
+    --mode simulation
```

Comment:
Good idea (we should do this more in functional tests), but be aware that simulation mode sleeps for some random interval unless configured otherwise (like we do in the integration tests), so may need to be backed up with a `[simulation]default run length = PT0S` config.